src/runner/bootstrapper.ts

```diff
@@ -171,8 +171,18 @@ export default class Bootstrapper {
         return BrowserSet.from(browserConnections, this._getBrowserSetOptions());
     }
 
-    private _filterTests (tests: Test[], predicate: Filter): Test[] {
-        return tests.filter(test => predicate(test.name as string, test.fixture.name as string, test.fixture.path, test.meta, test.fixture.meta));
+    private static async asyncFilter<T> (arr: T[], predicate: (item: T) => Promise<boolean>): Promise<T[]> {
+        const results = await Promise.all(arr.map(predicate));
+
+        return arr.filter((_v, index) => results[index]);
```

Comment:
The '_v' name for a variable looks strange. Please leave only '_' if you don't use this variable.

Reply:
done, thank you